WordPressAuthenticator/Signin/LoginEmailViewController.swift

```diff
@@ -559,6 +559,7 @@ extension LoginEmailViewController {
 
         // Disconnect now that we're done with Google.
         GIDSignIn.sharedInstance().disconnect()
+        WordPressAuthenticator.track(.signedIn, properties: ["source": "google"])
```

Comment:
Wasn't able to reach this code path.  I'm guessing its orphaned code now that the social options are available from the prologue vs the LoginEmailViewController.  @ScoutHarris, thoughts?

Reply:
That code is sort of feature flagged via the `showNewLoginFlow` flag in `WordPressAuthenticatorConfiguration`. I would suggest removing that flag and associated code in a separate PR. I'm pretty sure there's some storyboard mojo that needs to be removed as well. Yes, it's totally my fault - I haven't cleaned that up post SIWA launch.

Reply:
After further inspection, I think Woo is still using this code? They still show the Google login option on the email screen. So we can leave this for now I think.

Reply:
Good catch!